Misc/NEWS.d/3.13.0a5.rst

```diff
@@ -742,7 +742,7 @@ Add ``windows_31j`` to aliases for ``cp932`` codec
 .. nonce: fv35wU
 .. section: Library
 
-:func:`functools.partial`s of :func:`repr` has been improved to include the
+:func:`functools.partial`\s of :func:`repr` has been improved to include the
```

Comment:
Isn't this backwards?  Wasn't it the `repr` of `functools.partial` that was changed, rather than `partial`s of `repr`?
```suggestion
The :func:`repr` of :func:`functools.partial` objects has been improved to include the
```